### PR TITLE
Update graph_embeddings.py

### PR DESCRIPTION
**Pull request by bsantan**
search for exact embeddings filename to ensure dimensions match user specified value

### CODE REVIEW

File: graph_embeddings.py [hunk 1]
```diff
@@ -15,9 +15,7 @@ def __init__(self, triples_file,output_dir,input_dir,embedding_dimensions):
     def check_file_existence(self,embeddings_file):
         exists = 'false'
         for fname in os.listdir(self.output_dir):
-           # print(fname)
-            if bool(re.search("Embeddings", fname)):
-               # print('match')
+            if bool(re.search(embeddings_file, fname)):
```
Comment (lagillenwater) [inline graph_embeddings.py:18]: where is embeddings_file referenced?

Comment (bsantan) [reply graph_embeddings.py:18]: it's used in line 96 to read in the embeddings object as emb

Comment (lagillenwater) [reply graph_embeddings.py:18]: yep, I didn't expand the view enough. my bad.

**Review by lagillenwater — APPROVED** [review]
better logic if you start getting multiple embeddings files

**Pull request merged**